qiskit_experiments/library/tomography/fitters/cvxpy_lstsq.py

```diff
@@ -127,11 +131,32 @@ def cvxpy_linear_lstsq(
     Returns:
         The fitted matrix rho that maximizes the least-squares likelihood function.
     """
+    t_start = time.time()
+
     if measurement_basis and measurement_qubits is None:
         measurement_qubits = tuple(range(measurement_data.shape[1]))
     if preparation_basis and preparation_qubits is None:
         preparation_qubits = tuple(range(preparation_data.shape[1]))
 
+    input_dims, output_dims = _basis_dimensions(
```

Comment:
Seems like this is only used for generating metadata (this function uses this to create TP constraint though). How much this metadata is important?

Reply:
Ah okey this is used in the formatter.